src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -175,25 +194,37 @@ public static FormulaValue Text(EvalVisitor runner, EvalVisitorContext context,
                 }
             }
 
+            // We limit the format string size to 100 chars
+            if (formatString != null && formatString.Length > 100)
+            {
+                // !JYL! do we need a 'translatable' error message?
```

Comment:
>translatable [](http://example.com/codeflow?start=39&length=12)

there may be an existing one... or a generic "out or range" we can use.

Reply:
Haven't found it. Ended up creating a new one.